flink-connector-mysql-cdc/src/main/java/com/alibaba/ververica/cdc/connectors/mysql/table/MySQLTableSourceFactory.java

```diff
@@ -85,6 +85,11 @@
 			"MySQL database cluster as another server (with this unique ID) so it can read the binlog. " +
 			"By default, a random number is generated between 5400 and 6400, though we recommend setting an explicit value.");
 
+	private static final ConfigOption<Boolean> CAPTURE_UNCHANGED_UPDATES = ConfigOptions.key("capture-unchanged-updates")
+		.booleanType()
+		.defaultValue(true)
+		.withDescription("Optional flag whether to capture the updates if that have no change or not.default true, false to disable.");
```

Comment:
```suggestion
		.withDescription("Whether to capture the updates which don't change any monitored columns. This may happen if the monitored columns (columns defined in Flink SQL DDL) is a subset of the columns in database table. Default 'true', which means all the updates will be captured. You can set to 'false' to only capture unchanged updates, but note this may increase some comparison overhead for each update event.");
```

Reply:
great! but `set to 'false' to only capture unchanged updates` should be `set to 'false' to only capture changed updates`